src/components/VCard/VCard.ts

```diff
@@ -1,53 +1,75 @@
+// Styles
 import '../../stylus/components/_cards.styl'
 
+// Components
+import { VPaper } from '../VPaper'
+
 // Mixins
-import Colorable from '../../mixins/colorable'
 import Routable from '../../mixins/routable'
-import Themeable from '../../mixins/themeable'
 
 // Helpers
 import { convertToUnit } from '../../util/helpers'
-import mixins from '../../util/mixins'
 
 // Types
 import { VNode } from 'vue'
+import mixins from '../../util/mixins'
+import { deprecate } from '../../util/console'
 
 /* @vue/component */
-export default mixins(Colorable, Routable, Themeable).extend({
+export default mixins(
+  Routable,
+  VPaper
+).extend({
   name: 'v-card',
 
   props: {
+    elevation: {
+      type: [Number, String],
+      default: 1
+    },
     flat: Boolean,
     height: [Number, String],
-    hover: Boolean,
+    /* @deprecated */
     img: String,
+    /* @deprecated */
     raised: Boolean,
-    tag: {
```

Comment:
removing `tag` prop sounds like a breaking change

Reply:
this is in v-paper now